.github/workflows/main.yaml

```diff
@@ -167,31 +185,27 @@ jobs:
     if: ${{ always() }}
     runs-on: ubuntu-latest
     name: Build and test outcome
-    needs: [
-      build,
-      test-linux-distributions,
-      test-macos ]
+    needs: [ build, test-linux, test-macos ]
     steps:
-      - run: exit 1
-        if: >-
+      - if: >-
           ${{
                contains(needs.*.result, 'failure')
             || contains(needs.*.result, 'cancelled')
           }}
+        run: exit 1
```

Comment:
The way GitHub highlights this makes it look broken, but it looks fine in VS Code:
<img width="375" alt="Screenshot 2023-12-12 at 8 54 58 am" src="https://github.com/OctopusDeploy/OctopusClients/assets/25342760/47016c03-e1e8-4774-953d-bba290177469">